diesel/src/macros/mod.rs

```diff
@@ -45,6 +45,13 @@ macro_rules! column {
         {
         }
 
+        impl<ST, Source, Predicate> SelectableExpression<
+            $crate::query_source::filter::FilteredQuerySource<Source, Predicate>, ST>
+            for $column_name where
+            $column_name: SelectableExpression<Source, ST>
```

Comment:
Hm. This shouldn't be necessary actually, there should be a blanket `SelectableExpression` impl on `FilteredQuerySource`